iocore/net/I_NetVConnection.h

```diff
@@ -179,6 +179,10 @@ struct NetVCOptions {
 
   EventType etype;
 
+  /** ALPN protocol-lists. The format is OpenSSL protocol-lists format (vector of 8-bit length-prefixed, byte strings)
```

Comment:
@shinrich @a-canary What did you guys deal with ALPN for H2 origin cons? I think extending `NetVCOptions` is good place like SNI.